### PR TITLE
Add related to topic "Documentation"

### PR DESCRIPTION
**Pull request by shaedrich**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

Types of documentation

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/documentation/index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ display_name: Documentation
 short_description: Documentation is a set of information that describes a product
   to its users.
 topic: documentation
-wikipedia_url: https://en.wikipedia.org/wiki/Software_documentation
+wikipedia_url: https://en.wikipedia.org/wiki/Software_documentation,
```
Comment (ahpook) [inline topics/documentation/index.md:7]:
```suggestion
wikipedia_url: https://en.wikipedia.org/wiki/Software_documentation
```
You added an extra comma on this URL

**Review by ahpook — CHANGES_REQUESTED** [review]
1 change, otherwise LGTM

File: topics/documentation/index.md (hunk 1 above)
Comment (shaedrich) [reply topics/documentation/index.md]: Good catch!

**Review by ahpook — APPROVED** [review]
Nice!

**Comment by ahpook** [comment]
@shaedrich i just rebuilt the site, so all the changes you've made recently are live. Thanks so much for these contributions!

Screenshot 2023-10-13 at 5 05 17 PM

**Comment by shaedrich** [comment]
@ahpook You're very welcome&mdash;always a please to contribute!

Cool, I checked my changes and was wondering if the list parsing was intensional:
| Actual | Expected |
|-|-|
| grafik | grafik |
| grafik | grafik |

**Comment by kenyonj** [comment]
I think there is definitely a bug with the `ul`/`li` stuff, looks like we're not building those lists correctly from the markdown.
For the `git` topic page, I think it's treated differently because there is an associated event in the sidebar, GitHub Universe 2023. Whether or not that's the correct CSS to style that page, I'm not sure, but I think that's the reason that it's different.

**Comment by shaedrich** [comment]
Ah, yeah, makes sense.

**Comment by shaedrich** [comment]
But if you look at [the feed](https://explore-feed.github.com/feed.json), it's all inside one `<ul>`, however, delimited by `\n`

**Comment by shaedrich** [comment]
@kenyonj As far as I can see, that part is not open source so I can't fix that myself and create a PR, right?

**Comment by shaedrich** [comment]
Finally addressed that via https://github.com/orgs/community/discussions/86653

**Pull request merged**